### PR TITLE
Grey response fix

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### What's this PR do?
* Fix new story fields
* fix responses for posting users
* clean author seeds
#### Where should the reviewer start?
* the post and patch for users
#### How should this be manually tested?
create and patch users

### CODE REVIEW

**Review by carlymclift — APPROVED** [review]
Yay!

**Review by nickhartdev — APPROVED** [review]
Thanks for doing this!

File: db/seeds/dev/prompts_seed.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ exports.seed = function(knex) {
         },
         {
           id: 3,
-          prompt: "The president is a really nice guy",
+          prompt: "The president is a really nice guy (maybe too nice) and he calls you daily",
           genre: "satire",
```
Comment (Abdeboskey) [inline db/seeds/dev/prompts_seed.js:21]: Way to turn this one around

File: db/seeds/dev/authors_seed.js [hunk 2]
```diff
@@ -6,47 +6,54 @@ exports.seed = function (knex) {
...
+            id: 6,
+            name: 'Casey Dallavalle',
+            email: 'theceo@turing.net.net',
+            created_at: "2020-05-11T00:25:34.315Z",
+            updated_at: "2020-06-11T00:25:34.315Z",
+            password: 'qwerty',
+            bio: "Put a bird on it, and I'll be there."
           }
         ]);
       });
```
Comment (Abdeboskey) [inline db/seeds/dev/authors_seed.js:59]: This user info is CLUTCH

**Review by Abdeboskey — APPROVED** [review]
Awesome Greyson, thank you for this!

**Pull request merged**